pkg/cmd/release-payload-controller/cmd.go

```diff
@@ -57,12 +59,16 @@ func NewReleasePayloadControllerCommand(name string) *cobra.Command {
 func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.releaseNamespace, "release-namespace", o.releaseNamespace, "The namespace where ReleasePayloads are stored.")
 	fs.StringVar(&o.jobNamespace, "job-namespace", o.jobNamespace, "The namespace where release creation jobs are configured to run.")
+	fs.StringVar(&o.prowNamespace, "prow-namespace", o.prowNamespace, "The namespace where release creation jobs are configured to run.")
```

Comment:
I thought these were `jobs.batch`.  Why is this called a prow-namespace?

Reply:
Ultimately, we will be monitoring both `jobs.batch` and `prowjobs`.  This change was to get the correct information into the `prowjob_controller` for what is another PR that I'm working on (based off of this branch).

Reply:
This and the job-namespace have the same description, which is correct?

Reply:
This change has been completely removed from this PR (fixed in another, upcoming, PR).
Done